### ISSUE TITLE
Not able to get daily steps for a particular day

### ISSUE THREAD

**Issue by bluedragon14** [Stasonis/fitbit-api-example-android#2]
I've tried to modify the bindActivityData function to return the step value for a particular day.  I've tried the following:

```
Steps_ daily = activities.getBest().getTracker().getSteps();
daily.setDate("2016-12-18");
Integer dailySteps = daily.getValue();
```

I've also tried using tracker vs tracker_ and lifetime vs best to try and get different step values, but I am only able to get the number of steps for the first day w/ step values for a given fitbit account.

**Comment by toxic0n** [comment]
You will need to create your own wrapper, the one you are trying to use only returns Lifetime and Best totals.

Change your ActivityService.java URL to this:
```
  static DateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd");
  static String today = dateFormat.format(new Date());
  private final static String ACTIVITIES_URL = "https://api.fitbit.com/1/user/-/activities/date/"+ today +".json";
```
Add these lines to Activities.java:
` @SerializedName("summary")
    @Expose
    private Summary summary;

  public Summary getSummary() {
        return summary;
    }

`

Create  a Summary.java
```
public class Summary {

    @SerializedName("steps")
    @Expose
    private int steps;
    public int getSteps() {
        return steps;
    }
}
```
Then get your daily steps by:

 ` Integer dailySteps  = activities.getSummary().getSteps();`

**Comment by bluedragon14** [comment]
Thanks for the response!  I went a bit further and made a wrapper following the example response given at [https://dev.fitbit.com/docs/activity/#get-daily-activity-summary](url).  I have a class called Daily which consists of 3 classes, DailyActivity, Goals, and Summary.  I wrote them all similarly, following the style of stasonis' wrapper class, but here is the contents of Daily (DailyActivity, Goals and Summary follow the exact same style):

 `public class Daily {

    @SerializedName("dailyActivity")
    @Expose
    private DailyActivity dailyActivity;
    @SerializedName("goals")
    @Expose
    private Goals goals;
    @SerializedName("summary")
    @Expose
    private Summary summary;


    public DailyActivity getDailyActivity() { return dailyActivity; }
    public void setDailyActivity(DailyActivity dailyActivity) { this.dailyActivity = dailyActivity; }

    public Goals getGoals() { return goals; }
    public void setGoals(Goals goals) { this.goals = goals; }

    public Summary getSummary() { return summary; }
    public void setSummary(Summary summary) { this.summary = summary; }

}`

Now, I'm able to get daily steps by doing:
`Integer dailySteps = daily.getSummary().getSteps();`

But for some reason I am not able to get daily steps with this:
`Integer dailySteps = daily.getDailyActivity().getSteps();`

I noticed in the example response that the activities section has brackets.  Should I be doing something differently?  Also, what is the difference between the steps value in activities and summary?  If I'm interested in getting step values for the current day and want the value to update as the user syncs their Fitbit, is it okay to use the steps from summary?
